sdk/template/template/api-extractor.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "types/src/index.d.ts",
+  "docModel": {
+    "enabled": false
+  },
+  "apiReport": {
+    "enabled": true,
+    "reportFolder": "./review"
+  },
+  "dtsRollup": {
+    "enabled": true,
+    "untrimmedFilePath": "./types/azure-template.d.ts",
+    "betaTrimmedFilePath": "./types/azure-template-beta.d.ts",
+    "publicTrimmedFilePath": "./types/azure-template-public.d.ts"
```

Comment:
I think we only want "publicTrimmedFilePath". The others would be helpful if we released separate versions of the same library, which aren't currently planning on doing (instead it goes from preview to release directly)

Reply:
Make sense.  I will rename `untrimmedFilePath` to `./types/azure-template-dev.d.ts`. Otherwise it gets generated to `dist/template.d.ts` by default and is packed.  I noticed that eventhubs etc. are packing the untrimmed ones.  I will fix them in a separate PR.

Reply:
I'm trying to remember, you might also be able to set it to "" to not generate it.

Reply:
Fixed.